websites/G/Goodreads/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "charmines",
+    "id": "588110067656687740"
+  },
+  "url": [
+    "goodreads.com",
+    "www.goodreads.com"
+  ],
+  "description": {
+    "en": "Goodreads reading webpage to buy and browse online books"
+  },
+  "service": "Goodreads",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/FBlZvAu.png",

```

Comment:
get a better logo

Reply:
Just to further explain what I told you on discord, the image you used is the favicon which would appear as pixels on our site as it is made larger. 
https://i.imgur.com/BTqiVSE.png is a 512x512 image which would work instead!